standup/status/models.py

```diff
@@ -213,17 +230,7 @@ def htmlify(self):
             formatted = PULL_RE.sub(
                 r'<a href="%s/pull/\2">\1 \2</a>' % self.project.repo_url, formatted)
 
-        # Search for tags on the original, unformatted string. A tag must start
-        # with a letter.
-        tags = TAG_RE.findall(self.content)
-        tags.sort()
-        if tags:
-            tags_html = ''
-            for tag in tags:
-                tags_html = TAG_TMPL.format(tags_html, tag.lower(), tag)
-            formatted = '%s <div class="tags">%s</div>' % (formatted, tags_html)
-
-        return formatted
+        return Markup(MD.convert(formatted))
```

Comment:
I thought I pulled in some tests from the old standup related to formatting status posts, but you didn't edit any. So one of two things is possible:

1. Travis gets fussy with this PR.
2. We don't have these tests. If this is the case, I'll write up an issue and write some later.